modules/boltlib/lib/puppet/functions/run_task.rb

```diff
@@ -59,21 +46,27 @@ def run_task_instance(task, *targets)
       )
     end
 
+    use_args = task_args.nil? ? {} : task_args
+    # TODO: Is it OK to refer to the Puppet::Pal namespace here?
+    Puppet::Pal::TaskSignature.new(task).runnable_with?(use_args) do |mismatch|
```

Comment:
Is it OK to refer the `Puppet::Pal` namespace from the implementation of `run_task` puppet function provided by the `boltlib' module?